README.md

```diff
@@ -96,6 +96,7 @@ npm install @carbon/charts --save
 | Curved Line | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:
 | Pie         | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:
 | Step        | :white_check_mark: | :white_check_mark: | :white_check_mark: | :hourglass_flowing_sand:
```

Comment:
`Step` is available for all frameworks, could you change the last one to `:white_check_mark:` please

Reply:
Sure, I didn't want to touch anything that's not directly related to the issue